src/modules/privacy-idea-administration/privacy-idea-administration.service.spec.ts

```diff
@@ -79,6 +86,168 @@ export const mockNonErrorThrow = (): never => {
     throw { message: mockErrorMsg };
 };
 
+export const mockTokenVerificationResponse: TokenVerificationResponse = {
```

Comment:
Do these mock-data-objects need to be exported? 
Exporting things from test-files should not be done

Reply:
Removed